conversant/personas/historian/config.json

```diff
@@ -0,0 +1,52 @@
+{
+    "chatbot_config": {
+        "max_context_examples": 10,
+        "avatar": ":amphora:"
+    },
+    "client_config": {
+        "model": "command-52b-v5-dec22-eeayj60s",
```

Comment:
Can we change this model to `command-xlarge-20221108`, the latest command model that is also in our playground?

Reply:
Done!